Cargo.toml

```diff
@@ -13,8 +13,15 @@ readme = "README.md"
 [lib]
 path = "lib.rs"
 
+[features]
+eserde = ["erased-serde"]
```

Comment:
I wish this was just `serde`, and not `eserde`.

Reply:
Unfortunately it can't be `serde` because we also depend on real `serde`.  If you try it:

```
error: failed to parse manifest at json/Cargo.toml`

Caused by:
  Features and dependencies cannot have the same name: `serde`
```

Happy to change to something else if you prefer?

Reply:
Maybe something describing functionality, then? `nested-values` or something like that.

Reply:
We probably should make it the same here and in other crates.

Reply:
Done.